R/diagnostics.R

```diff
@@ -19,6 +19,10 @@
 #' @param fit A fitted model of class `epidist_fit`
 #' @family diagnostics
 #' @autoglobal
+#' @importFrom brms nuts_params rhat
```

Comment:
following the package recs shouldn't these just be package::function as they are only used in this one function? Or are we going with a policy of everything in the namespace?

Reply:
Yeah OK I should actually read the package book

Reply:
I mean just that one little section!